src/main/docker/artemis-server-client-mysql.yml

```diff
@@ -0,0 +1,62 @@
+# ----------------------------------------------------------------------------------------------------------------------
+# Artemis-Server-Client-MySQL Setup (CURRENTLY NOT MAINTAINED)
+# ----------------------------------------------------------------------------------------------------------------------
+
+# NOTE: this docker compose file starts the artemis-server (as jar file) and the artemis-client in separate containers.
+# This setup is aimed for development.
+# If you want to start the whole Artemis application (server and client) in the same container, you need to use
+# the Artemis-Dev-MySQL Setup contained in artemis-dev-mysql.yml
+
+# TODO: unify more into depth for separate JS and Java container (maybe even sperated Dockerfiles?) for development?
+# TODO: or remove this setup completely
```

Comment:
Shall we remove this setup variant as it's pretty outdated and not used? Or keep it with the TODOs?

Reply:
To ease maintenance, I’d say removing it is fine if its not used anyway.

Reply:
Yeah, I had the same feeling, just wanted to have another opinion on this. Thx! If someone else feels like this should stay they can post it here.